src/10-neb/broker.cc

```diff
@@ -63,9 +64,7 @@ void broker_module_init(void const* arg) {
     if (neb_category != io::events::neb) {
       e.unregister_category(neb_category);
       --neb_instances;
-      throw(exceptions::msg() << "NEB: category " << io::events::neb
-                              << " is already registered whereas it should be "
-                              << "reserved for the NEB module");
+      throw(exceptions::msg_fmt("NEB: category '{}' is already registered whereas it should be reserved for the NEB module", io::events::neb));
```

Comment:
Please, throw is not a function so no parenthesis.
throw exceptions::... is better.